### PR TITLE
[support] Replace 'could' and 'might'

### PR DESCRIPTION
**Pull request by jensmaurer**
as directed by ISO/CS.

Partially addresses #4319

### CODE REVIEW

File: source/support.tex [hunk 1]
```diff
@@ -542,7 +542,7 @@
 after inclusion of any member of the set of library headers
 indicated in the corresponding comment in this synopsis.
 \begin{note}
-Future revisions of \Cpp{} might replace
+Future revisions of \Cpp{} can replace
```
Comment (jwakely) [inline source/support.tex:545]: I know it's necessary, but this does not seem like an improvement.

File: source/support.tex [hunk 2]
```diff
@@ -1948,13 +1948,13 @@
...
-The order of registration could be indeterminate if \tcode{at_quick_exit} was called from more
-than one thread.
+The order of registration can be indeterminate
+if \tcode{at_quick_exit} is called from more than one thread.
 \end{note}
 \begin{note}
 The
 \tcode{at_quick_exit} registrations are distinct from the \tcode{atexit} registrations,
-and applications might need to call both registration functions with the same argument.
+and applications can call both registration functions with the same argument.
```
Comment (jwakely) [inline source/support.tex:1957]: This seems to change the meaning. I know it's effectively the same, but it imparts different information to the reader.

File: source/support.tex (hunk 1 above)
Comment (CaseyCarter) [reply source/support.tex:545]: How about rephrasing completely: "The values of these macros will increase in future revisions of \Cpp{} which modify the corresponding feature."?

Comment (jensmaurer) [reply source/support.tex:545]: Future revisions of \Cpp{} will replace
the values of these macros with greater values
when the corresponding feature is modified.

File: source/support.tex (hunk 2 above)
Comment (jensmaurer) [reply source/support.tex:1957]: Tried a different approach.

**Review by jwakely — APPROVED** [review]

File: source/support.tex [hunk 3]
```diff
@@ -2236,7 +2237,7 @@
 the program shall also define
 the corresponding version without the \tcode{size} parameter.
 \begin{note}
-The default behavior below might change in the future, which will require
+The default behavior below can change in the future, which will require
 replacing both deallocation functions when replacing the allocation function.
```
Comment (zygoloid) [inline source/support.tex:2241]: I think this has lost the intent, which is that we've identified this as a desirable change. How about:
```suggestion
It is anticipated that the default behavior will change in the future and will instead require
replacing both deallocation functions when replacing the allocation function.
```

File: source/support.tex [hunk 4]
```diff
@@ -2491,7 +2492,7 @@
 the program shall also define
 the corresponding version without the \tcode{size} parameter.
 \begin{note}
-The default behavior below might change in the future, which will require
+The default behavior below can change in the future, which will require
```
Comment (zygoloid) [inline source/support.tex:2495]:
```suggestion
It is anticipated that the default behavior will change in the future and will instead require
```

**Comment by jensmaurer** [comment]
applied, squashed + force-pushed

**Pull request merged**